### PR TITLE
Cartooning Image

### LINKED ISSUE #559: Cartooning an image with openCV
Computer Vision as you know (or even if you don’t) is a very powerful tool with immense possibilities. So, when I set up to prepare a comic of one of my friend’s college life, I soon realized that I needed something that would reduce my efforts of actually painting it but will retain the quality and I came up with the following solution. So, in this project we will cartoonize an image with openCV.

Please assign me this task under gssoc.

### PR DESCRIPTION
**Pull request by aliya-rahmani**
Description
Cartooning image using OpenCV
Fixes #559 
Have you read the Contributing Guidelines on Pull Requests?
Yes

### CODE REVIEW

File: Crossword Solver/xword.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```
Comment (kaustubhgupta) [inline Crossword Solver/xword.py:1]: Delete this file, it's not related to the current issue.

File: Cartooning Image/cartooneffect.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import cv2 
+import numpy as np 
+  
+# reading image  
+img = cv2.imread("koala.jpeg")
```
Comment (kaustubhgupta) [inline Cartooning Image/cartooneffect.py:5]: Take the input of the file name from the user (Full path)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Add readme of the script along with these changes:

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Did you run the script after updating it? Also delete the file requested and add readme file

File: Cartooning Image/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Cartooning Image with OpenCV
```
Comment (kaustubhgupta) [inline Cartooning Image/README.md:1]: After this heading, maintain the levels of all subheadings as (##)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by kaustubhgupta — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]
LGTM 👍🏻

**Review by santushtisharma10 — APPROVED** [review]

**Comment by aliya-rahmani** [comment]
@kaustubhgupta, Thanks for guiding me all the way!

**Review by avinashkranjan — APPROVED** [review]
LGTM..

**Pull request merged**